media-gfx/maim/maim-5.7.4.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Commandline tool to take screenshots of the desktop"
+HOMEPAGE="https://github.com/naelstrof/maim"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/naelstrof/maim.git"
+else
+	SRC_URI="https://github.com/naelstrof/maim/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="icu"
+
+DEPEND="
+	media-libs/libglvnd
+	media-libs/libjpeg-turbo
+	media-libs/libpng:0=
+	x11-libs/libX11
+	x11-libs/libXcomposite
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXrandr
+	x11-libs/libXrender
+	>=x11-misc/slop-7.5:=
+	icu? ( dev-libs/icu:= )"
```

Comment:
This also needed some updates, 
```
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find GLX (missing: GLX_INCLUDE_DIR GLX_LIBRARY)
```
for me this was fixed by installing *virtual/opengl*. 

And:
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
WEBP_INCLUDE_DIR (ADVANCED)
```
which obviously came from *media-libs/libwebp*. Always study the build system requirements and configure options when making a new version bump!